### ISSUE TITLE
CNN from scratch : tuning hyperparamètres

### ISSUE THREAD

**Issue by Calamia92** [Calamia92/dog-classifier#8]
Expérimenter learning rate, batch size, nombre d’époques.

Checklist

- [ ]  Grid search ou random search
- [ ]  Tableau comparatif des résultats
- [ ]  Sélection du meilleur set
